db/migrate/20180329174320_create_books_categories.rb

```diff
@@ -0,0 +1,11 @@
+class CreateBooksCategories < ActiveRecord::Migration
+  def change
+    create_table :books_categories do |t|
+      
+      t.integer :book_id
+      t.integer :category_id
+
+      t.timestamps null: false
```

Comment:
Are you sure you need a timestamp field on the `books_categories` table? The example in section 2.6 here http://guides.rubyonrails.org/association_basics.html doesn't have it, so unless you had a specific plan for it, it's probably not needed.